category.php

```diff
@@ -10,45 +10,36 @@
  * @since 1.2.1
  */
 
-get_header(); ?>
+get_header();
+
+$sidebar_class = '';
+if ( is_active_sidebar( 'sidebar-1' ) ) {
+	$sidebar_class = 'has-sidebar';
+}
+?>
 
 <div id="stage" class="inner" role="main">
-	<div id="content" class="content has-sidebar">
+	<div id="content" class="content <?php echo esc_attr( $sidebar_class ); ?>">
 
 		<div class="content-main main-content">
-		<?php if ( have_posts() ) : ?>
-			<header class="archive-header">
-				<h1 class="archive-title">
-					<?php printf( 'Category Archives: %s', '<span>' . single_cat_title( '', false ) . '</span>' ); ?>
-				</h1>
-
-			<?php if ( category_description() ) : // Show an optional category description. ?>
-				<div class="archive-meta"><?php echo category_description(); ?></div>
+
+			<?php if ( have_posts() ) : ?>
+				<header class="archive-header">
+					<h1 class="archive-title">
+						<?php printf( 'Category Archives: %s', '<span>' . single_cat_title( '', false ) . '</span>' ); ?>
+					</h1>
+				</header><!-- .archive-header -->
 			<?php endif; ?>
-			</header><!-- .archive-header -->
 
 			<?php
-			/* Start the Loop */
-			while ( have_posts() ) : the_post();
-
-				/*
-                 * Include the post format-specific template for the content. If you want to
-                 * this in a child theme then include a file called called content-___.php
-                 * (where ___ is the post format) and that will be used instead.
-				 */
-				get_template_part( 'content', get_post_format() );
-
-			endwhile;
-
-			twentytwelve_content_nav( 'nav-below' );
+			/* We removed the loop because we don't use this display template. */
```

Comment:
That's very helpful context for this review, but I'm not sure it needs to remain in the codebase.

Reply:
This is as much a note to future-me when I end up looking at this template and wondering why The Loop isn't there. I've got a ticket to revisit this, though - it may be possible to go deeper with a more systematic removal of unneeded features.